packages/edit-site/src/components/sidebar-edit-mode/template-panel/index.js

```diff
@@ -58,12 +64,10 @@ export default function TemplatePanel() {
 			>
 				<TemplateAreas />
 			</SidebarCard>
-			<PanelRow
```

Comment:
Is this change necessary?

Reply:
The empty row creates a bunch of empty space in the sidebar when it shouldn't be there as I noted in my [review](https://github.com/WordPress/gutenberg/pull/54640#pullrequestreview-1634713475)

Reply:
Oh yeah I just saw that now 😅 . 👍